src/v/redpanda/admin_server.cc

```diff
@@ -3310,6 +3310,21 @@ void admin_server::register_debug_routes() {
           return ss::make_ready_future<ss::json::json_return_type>(
             _metadata_cache.local().is_node_isolated());
       });
+
+    register_route<user>(
+      seastar::httpd::debug_json::get_controller_status,
+      [this](std::unique_ptr<ss::httpd::request>)
+        -> ss::future<ss::json::json_return_type> {
+          vlog(logger.info, "Request to get controller info");
```

Comment:
the admin API already logs a message at debug level for each request (in `log_request`), so you shouldn't need this one